Cargo.toml

```diff
@@ -70,4 +70,7 @@ debug = 1
 debug-assertions = false
 panic = 'unwind'
 incremental = true
-codegen-units = 256
\ No newline at end of file
+codegen-units = 256
+
+[profile.release.package.mocktopus]
+opt-level = 1 # TODO: MIR fails on optimizing this dependency, remove that..
```

Comment:
mocktopus should be a dev only dependency, but since `TestCoin` is used outside coins crate I will use `for-tests` feature instead. Will push a commit that fixes that.

Reply:
Fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2149/commits/db8aedcba0eff5b56a776f92e5985d9429f42dac

In the future, we should move `TestCoin` to `mm2_test_helpers` or another test only crate that we import as a dev only dependency.

Reply:
I will revert the last commit for now, I thought everything was fine because clippy for all targets and features worked locally and also tests but for CI jobs it's different. It will need a bigger refactor.

Reply:
IMO we should just get rid of that outdated/dead dependency.

Reply:
> BTW I made mocktopus optional in my PR https://github.com/KomodoPlatform/komodo-defi-framework/pull/2112 too.
But I had to add it also as dev-dependencies as some tests in mm2_main rely on mocks in coins: https://github.com/KomodoPlatform/komodo-defi-framework/commit/f3625e5ad9ec5eec4a86b61fff475be963551a19 (otherwise mocks did not do anything)

Thanks @dimxy for this, I guess we can merge this with `[profile.release.package.mocktopus]` and you can remove it in your PR if you can to not duplicate the effort.